sdks/java/core/src/main/java/org/apache/beam/sdk/testing/CombineFnTester.java

```diff
@@ -102,6 +102,7 @@
       } else {
         accumulator = fn.mergeAccumulators(Arrays.asList(accumulator, inputAccum));
       }
+      fn.extractOutput(accumulator); // Extract output to simulate multiple firings.
```

Comment:
Hm don't understand this comment. Is there a way to more directly simulate multiple firings, and assert that the results of each firing are correct?

Reply:
I wanted to invoking fn.extractOutput() multiple times as it seems to happen on the runners like Dataflow when there are multiple firings. I am not sure if there is an easy way to add validation for each firing here. Since the final extractOutput() is verified, intermediate values have some indirect validation. Without this, the test added in `ApproximateUniqueTest` passes even without the fix.

Reply:
Do you have any suggestions for improving this?